source/_integrations/azure_data_explorer.markdown

```diff
@@ -0,0 +1,188 @@
+azure_data_explorer:
+  filter:
+    include_domains:
+    - homeassistant
+    - light
+    - media_player
+```
+
+{% configuration %}
+filter:
+  description: Filter domains and entities for Data Explorer. ([Configure Filter](#configure-filter))
+  required: true
+  type: map
+  default: Includes all entities from all domains
+  keys:
+    include_domains:
+      description: List of domains to include (e.g., `light`).
+      required: false
+      type: list
+    exclude_domains:
+      description: List of domains to exclude (e.g., `light`).
+      required: false
+      type: list
+    include_entity_globs:
+      description: Include all entities matching a listed pattern (e.g., `sensor.weather_*`).
+      required: false
+      type: list
+    exclude_entity_globs:
+      description: Exclude all entities matching a listed pattern (e.g., `sensor.weather_*`).
+      required: false
+      type: list
+    include_entities:
+      description: List of entities to include (e.g., `light.attic`).
+      required: false
+      type: list
+    exclude_entities:
+      description: List of entities to include (e.g., `light.attic`).
+      required: false
+      type: list
+{% endconfiguration %}
+
+<div class='note warning'>
+Not filtering domains or entities will send every event to Azure Data Explorer.
+</div>
+
+
+### Configuring a filter
+
+By default, no entity will be excluded. To limit which entities are being exposed to `Azure Data Explorer`, you can use the `filter` parameter.
+
+```yaml
+# Example filter to include specified domains and exclude specified entities
+azure_data_explorer:
+  filter:
+    include_domains:
+      - alarm_control_panel
+      - light
+    include_entity_globs:
+      - binary_sensor.*_occupancy
+    exclude_entities:
+      - light.kitchen_light
+```
+
+Filters are applied as follows:
+
+1. No includes or excludes - pass all entities.
+2. Includes, no excludes - only include specified entities.
+3. Excludes, no includes - only exclude specified entities.
+4. Both includes and excludes:
+   - Include domain and/or glob patterns specified:
+      - If the domain is included, and the entity is not excluded or matches the exclude glob pattern, pass.
+      - If the entity matches include glob pattern, and the entity does not match any exclude criteria (domain, glob pattern, or listed), pass.
+      - If the domain is not included, the glob pattern does not match, and the entity not included, fail.
+   - Exclude domain and/or glob patterns specified and include does not list domains or glob patterns
+      - If domain is excluded and entity not included, fail
+      - If entity matches exclude glob pattern and entity not included, fail
+      - If entity does not match any exclude criteria (domain, glob pattern or listed), pass
+   - Neither include or exclude specifies domains or glob patterns
+      - If entity is included, pass (as #2 above)
+      - If entity include and exclude, the entity exclude is ignored
```

Comment:
Not sure I understand. Is this *If the entity matches both an include and an exclude pattern*?

Reply:
Correct 👍